src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -503,6 +503,17 @@ void Renderer::Impl::render(const UpdateParameters& updateParameters) {
     }
 #endif
 
+    // Ends the RenderPass
+    parameters.renderPass.reset();
```

Comment:
what is the reason it cannot die after `parameters.encoder->present(parameters.backend.getDefaultRenderable());`?

Reply:
Clarified offline, it is about order.